src/ODataHttpClient.Tests/Factories/HeadersFactory.cs

```diff
@@ -0,0 +1,30 @@
+using System.Collections.Generic;
+using System.Net.Http;
+using System.Net.Http.Headers;
+
+namespace ODataHttpClient.Tests.Factories
+{
+    public static class HeadersFactory
```

Comment:
I think that this is not needed if Request#Headers's type is IDicitonary<string, string>.

Reply:
@iwate, type remains the same HttpRequestHeaders.